submissions/xT0PERx/Friends app/js/index.js

```diff
@@ -0,0 +1,175 @@
+const friendsList = {
+  current: [],
+  changed: [],
+  count: 72,
+};
+
+const ALL_USERS_CARDS_FILTERS = friendsList.changed;
+const ALL_USERS_CARDS = friendsList.current;
+const USERS_LIST = document.getElementById("usersList");
+const FILTERS_MENU = document.getElementById("filters_menu");
+let WHAT_FILTER_CHOSEN = "";
+const SEARCH_INPUT = document.querySelector(".searcher");
+
+function initApp() {
+  getUsers().then((users) => {
+    saveUsers(users);
+    createUsersCards(ALL_USERS_CARDS);
+  });
+}
+
+initApp();
+
+function getUsers() {
+  const url = `https://randomuser.me/api/1.3/?results=${friendsList.count}`;
+
+  return fetch(url)
+    .then((res) => {
+      if (res.ok) {
+        return res.json();
+      }
+    })
+    .then((data) => data.results)
+    .catch((err) => {
+      console.error(err.message);
+    });
+}
+
+function saveUsers(users) {
+  users.forEach(function (el) {
+    ALL_USERS_CARDS.push(el);
+    ALL_USERS_CARDS_FILTERS.push(el);
+  });
+}
+
+function createUsersCards(users) {
+  const lists = document.createDocumentFragment();
+
+  users.forEach(function (el) {
+    const card = document.createElement("li");
+    card.classList.add("userCard");
+    template = `<span class="name">${el.name.first} ${el.name.last}</span>
+     <img src="${el.picture.large}">
+     <span class="age">Age: ${el.dob.age}</span>
+     <span class="location">Location: ${el.location.city}</span>`;
+    card.insertAdjacentHTML("beforeEnd", template);
+    lists.append(card);
+  });
+
+  USERS_LIST.appendChild(lists);
+}
+
+FILTERS_MENU.addEventListener("click", function ({ target }) {
+  if (target.nodeName === "INPUT") {
```

Comment:
Since my __input__ with the __label__ is considered __one target__, nothing else comes to my mind than this check. I would be glad to your advice. Thank.